server/routes/user_screen/create.js

```diff
@@ -0,0 +1,38 @@
+const userScreen = require("../../query/user_screen");
+const express = require("express");
+const router = express.Router();
+
+router.post("/", async (req, res) => {
+  try {
+    const { screen_canvas_id, user_billboard_id, user_screen_name, description, config } = req.body;
+
+    let params = {};
+    if (screen_canvas_id) {
+      params.screen_canvas_id = screen_canvas_id;
+    }
+    if (user_billboard_id) {
+      params.user_billboard_id = user_billboard_id;
+    }
+    if (user_screen_name) {
+      params.user_screen_name = user_screen_name;
+    }
+    if (description) {
+      params.description = description;
+    }
+    if (config) {
+      if (typeof config === "object") {
+        params.config = JSON.stringify(config);
+      } else {
+        params.config = config;
+      }
+      
+    }
+    
+    const response = await userScreen.create(params);
+    res.json(response);
+  } catch (error) {
+    res.json(error);
```

Comment:
did you want to add a console.error message here too?

Reply:
In the future do we want to parse the errors and send more client friendly error messages?

This is the error message I got when I tried to use end-point with some broken data and it was not obvious to me 
how to resolve this.

```
{
    "length": 104,
    "name": "error",
    "severity": "ERROR",
    "code": "22P02",
    "position": "111",
    "file": "uuid.c",
    "line": "134",
    "routine": "string_to_uuid"
}
```

Reply:
That's true, for readability this would be helpful. Lets also make a separate ticket for this and have this and other other ticket above be minor priorities for now since we want to focus on making the features first. If any one has time they can grab these ticket at the same time for their next pull request.

Reply:
@richardfigueroa i totally agree. those error messages are the worst error messages possible lol